lib/app/modules/loan_application/controllers/loan_application_controller.dart

```diff
@@ -21,7 +19,7 @@ class LoanApplicationController extends GetxController {
   final loanAmountController = TextEditingController();
   final termController = TextEditingController();
 
-  Future<void> loanLogic() async {
+  void loanLogic() async {
```

Comment:
Please rename this function. Functions should have a verb, explaining what they do.